roles/openshift-applier/tasks/main.yml

```diff
@@ -15,6 +15,12 @@
   when:
   - ansible_connection != 'local'
 
+- name: "Change 'params_from_vars' fact into command line parameters"
+  set_fact:
+    oc_param_option: "{{ oc_param_option | default('') }} --param={{ item.key }}={{ item.value }}"
```

Comment:
should we make it `--param={{ item.key | upper }}` so we can suppport more ansible-y lowercase dictionary terms?

Reply:
No - parameters can be any case, so if we do the `| upper` we're ruling out using 3rd party templates that have params set with a mixed-case.

Reply:
I agree that for the most part templates use all UPPERCASE, but they don't have to and hence don't want to lock in the applier.